### PR TITLE
[IMP] Define jPdfSign max heap size

### PR DESCRIPTION
**Pull request by antespi**
This PR maybe fix memory problems in runbot (and other servers too)

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

**Comment by antespi** [comment]
jejeje, it runs well !! I got it!

**Comment by antespi** [comment]
@gurneyalex this PR fix memory problem at runbot in this addon, please review it, thanks

**Comment by carlosdauden** [comment]
:+1:

**Comment by sergio-teruel** [comment]
:+1:

File: report_qweb_signer/models/report.py [hunk 1]
```diff
@@ -145,7 +147,8 @@ def pdf_sign(self, pdf, certificate):
             raise Warning(
                 _('Signing report (PDF)'),
                 _('jPdfSign failed (error code: %s). '
-                  'Message: %s') % (str(process.returncode), err))
+                  'Message: %s. Output: %s') %
+                (str(process.returncode), err, out))
```
Comment (yajo) [inline report_qweb_signer/models/report.py]: [`str()` is implicit with `%s`](https://docs.python.org/2/library/stdtypes.html#string-formatting-operations) AFAIK.

Comment (pedrobaeza) [inline report_qweb_signer/models/report.py]: Yeah, that is

**Comment by rafaelbn** [comment]
:+1:

File: report_qweb_signer/models/report.py [hunk 2]
```diff
@@ -145,7 +147,8 @@ def pdf_sign(self, pdf, certificate):
             raise Warning(
                 _('Signing report (PDF)'),
```
Comment (pedrobaeza) [inline report_qweb_signer/models/report.py]: Warning only uses one argument, not 2 as the old `except_orm`

Comment (antespi) [inline report_qweb_signer/models/report.py]: But It's not what i see here: https://github.com/odoo/odoo/blob/8.0/openerp/exceptions.py#L33
isn't it?

File: report_qweb_signer/models/report.py (hunk 1 above)
Comment (pedrobaeza) [inline report_qweb_signer/models/report.py]: Warning inherits from Exception, not from except_orm

File: report_qweb_signer/models/report.py (hunk 2 above)
Comment (pedrobaeza) [inline report_qweb_signer/models/report.py]: Warning inherits from Exception, not from except_orm

Comment (antespi) [inline report_qweb_signer/models/report.py]: After a depth looking... you're right :smile:

**Comment by antespi** [comment]
@Yajo, @pedrobaeza fixes suggested done, thanks.

**Comment by yajo** [comment]
:+1:

**Pull request merged**